src/main/java/com/ericsson/ei/services/SubscriptionService.java

```diff
@@ -207,4 +220,27 @@ private String generateQuery(String subscriptionName, String ldapUserName) {
         }
         return query;
     }
+
+
+    /**
+     * This method finds whether a given subscription has an owner
+     *
+     * @param subscriptionName-
+     *            subscription name
+     * @return a boolean
+     * @throws SubscriptionNotFoundException
+     */
+
```

Comment:
There should not be a line between javadock and function

Reply:
done